gravitee-apim-console-webui/src/management/api/analytics/logs/configuration/api-logs-configuration.component.html

```diff
@@ -0,0 +1,96 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2>
+  Logging configuration
+  <button mat-raised-button type="button" class="go-to-dashboard" aria-label="Go to Logging dashboard" (click)="goToLoggingDashboard()">
+    Go to Logging dashboard
+  </button>
+</h2>
+
+<gio-banner-info class="logging-banner">
+  <span>
+    Use logging with caution and only for development purpose. It can have an impact on API performances and is requiring more spaces for
```

Comment:
I think "purpose" should be plural but i'm not sure. @ivan-gravitee any thought ?

Reply:
Also, the "is requiring" does not feel natural to me

Reply:
Correct @a-cordier:

Use logging with caution and only for development purposes. It requires more space for analytics storage and can impact API performance.

Reply:
@ivan-gravitee - yes, I think that looks good.

Reply:
Thanks @AndrewMcFarlandCampbellGravitee

Reply:
Done